File: src/levels/L10-observer/base_cocina.rb
class Order
  def initialize(display, push, kitchen)
    @display = display
    @push = push
    @kitchen = kitchen # ← otro interesado, otra dependencia
  end

  # region: Order#complete!
  def complete!
    @status = :listo
    @display.update(self)
    @push.update(self)
    @kitchen.update(self) # ← Order modificado otra vez
  end
  # endregion
end

File: src/levels/L10-observer/ts/base_cocina.ts
class Order {
  private status = 'pendiente'

  constructor(
    private display: Display,
    private push: PushNotifier,
    private kitchen: Kitchen, // ← otro interesado, otra dependencia
  ) {}

  // region: Order#complete!
  complete() {
    this.status = 'listo'
    this.display.update(this)
    this.push.update(this)
    this.kitchen.update(this) // ← Order modificado otra vez
  }
  // endregion
}
